tasks/main.yml

```diff
@@ -12,9 +12,9 @@
 
 # The netq package installs addtional config to rsyslog
 - name: Install netq agent
-  apt:
+  package:
```

Comment:
If we want the role to support installing netq on more than just cumulus, the version check above will need modification as it only targets cumulus.

Reply:
Indeed, thank you for reminding me of that. I will think about how to solve this.

Reply:
Well, out of scope I think, and since then there is now a Netq3 branch I want to introduce that would change some of how the Ubuntu and CentOS stuff would be done.

Reply:
I'll create a ticket/issue for adding CentOS/Ubuntu support.